llvm/test/CodeGen/SPARC/salvage-debug-isel.ll

```diff
@@ -0,0 +1,85 @@
+; RUN: llc -march=sparc -O1 %s -o - | FileCheck %s
+
+; Debug info salvaging in isel means we should see a location for this variable.
+
+; CHECK-LABEL: a:
+; CHECK:    !DEBUG_VALUE: a:d <- [DW_OP_plus_uconst 98, DW_OP_plus_uconst 3, DW_OP_stack_value] $o6
+
+define dso_local zeroext i16 @a() local_unnamed_addr #0 !dbg !7 {
+entry:
+  %b = alloca [6 x i8], align 1, !DIAssignID !24
+    #dbg_assign(i1 undef, !14, !DIExpression(), !24, ptr %b, !DIExpression(), !25)
+  call void @llvm.lifetime.start.p0(i64 6, ptr nonnull %b) #2, !dbg !26
```

Comment:
Do we need the lifetime markers for the test to work? If not, please can you remove them (and the lifetime intrinsic declarations)?